core/templates/dev/head/css/oppia.css

```diff
@@ -2215,7 +2215,7 @@ pre.oppia-pre-wrapped-text {
   position: absolute;
   top: 0;
   width: 100%;
-  z-index: 10;
+  z-index: -10;
```

Comment:
Is this intentional (negative z-index for oppia-editable-section-mask)? @bansalnitish 
(reason behind all test failures `.oppia-editable-section-mask` not visible).

Reply:
I'll have a look once again on this value and set it to something positive if possible.